docs/intro/import-guide.rst

```diff
@@ -65,7 +65,7 @@ Building your documentation
 ---------------------------
 
 Within a few seconds of completing the import process,
-your code will automatically be fetched from your public repository,
+your code will automatically be fetched from your repository,
```

Comment:
We already mention in the beginning of the doc that users should use .org for public repos and .com for private repos.